test/Microsoft.Framework.Runtime.Tests/TestSample/project.lock.sample

```diff
@@ -0,0 +1,2180 @@
+{
```

Comment:
It won't. The name is not `project.lock.json`. It's a simple text file serve as a test resource.